app/router.js

```diff
@@ -7,7 +7,7 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders-list', { path: '/' })
+  this.route('reminders-list', { path: '/reminders' })
```

Comment:
The route here should simply be 'reminders', the path should be '/'. This indicates that when the path matches '/',  ember will look for the `reminders` route which will hand it the model it needs to render.